openmetadata-ui/src/main/resources/ui/playwright/utils/tag.ts

```diff
@@ -51,20 +54,54 @@ export const visitClassificationPage = async (
   await expect(page.locator('.activeCategory')).toContainText(
     classificationName
   );
+
+  await page.waitForSelector('[data-testid="loader"]', { state: 'detached' });
 };
 
-export const addAssetsToTag = async (page: Page, assets: EntityClass[]) => {
+// Other asset type that should not get from the search in explore, they are not added to the tag
+export const addAssetsToTag = async (
+  page: Page,
+  assets: EntityClass[],
+  tag: TagClass,
+  otherAsset?: EntityClass[]
+) => {
+  const res = page.waitForResponse(`/api/v1/tags/name/*`);
+  await tag.visitPage(page);
+  await res;
+
   await page.getByTestId('assets').click();
+  const initialFetchResponse = page.waitForResponse(
+    '/api/v1/search/query?q=&index=all&from=0&size=25&deleted=false**'
+  );
```

Comment:
this is fine for now, but this can lead to unnecessary failure if any of the data change